addon/templates/components/frost-bunsen-input-checkbox-array.hbs

```diff
@@ -0,0 +1,22 @@
+<label class={{labelWrapperClassName}}>
+  {{renderLabel}}
+  {{#if required}}
+    <div class='required'>Required</div>
+  {{/if}}
+</label>
+<div class={{inputWrapperClassName}}>
+  {{#each options as |option|}}
+    {{frost-checkbox
+      disabled=disabled
+      onInput=(action 'handleChange')
+      size=size
```

Comment:
In order for this to work you need to add the following to your component JavaScript.

```js
  @readOnly
  @computed('cellConfig')
  size (cellConfig) {
    return _.get(cellConfig, 'renderer.size') || 'medium'
  },
```